examples/running_log/main.go

```diff
@@ -15,12 +15,7 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	defer func() {
```

Comment:
View this doesn't seem right. Nor has it is necessary to test, lint is good practice